### DISCUSSION TITLE
Wlan Flsun 400V Install Klipper

### DISCUSSION THREAD

**Discussion by SirVaderx** [Guilouz/Klipper-Flsun-Speeder-Pad#185] [General]
Hello everyone,

I have a problem with my WiFi. When installing I have to choose my WiFi and password. What am I doing wrong here ( nmcli device wifi connect "Your_SSID" password "Your_Password"

I enter it like this ( nmcli device wifi connect "FRITZ!Box7590FT" password "99218811357247886716" )

What is wrong and how should I enter it correctly?

Greetings Markus

Sorry for my English

**Comment by Hydra20vt** [comment]
I have encountered the same problem yestarday during a fresh install.
Try this command instead:

sudo nmcli --ask dev wifi connect Your_ssid

then you will be asked for the password and you should be able to connect to the network

**Comment by SirVaderx** [comment]
Hello, thank you very much, I'll try it out tomorrow afternoon after
work. I'll let you know if it worked.

Thanks
Greetings Markus


Am Mi., 4. Sept. 2024 um 14:53 Uhr schrieb Hydra20vt <
***@***.***>:

**Reply by SirVaderx** [reply]
Hello, if I display the networks, from then on I can't do anything. At the bottom it says end and I can copy and paste and then it just says return, I don't know what to do next. I'll send the picture with the one at the top is mine Fritz Box.

Greetings Markus

**Comment by SirVaderx** [comment]
![20240906_141300](https://github.com/user-attachments/assets/34399bca-3a99-4e10-9001-b3911dd8a961)

**Reply by SirVaderx** [reply]
So I've now followed the instructions exactly as described. Under the section nmcli device wifi connect "Your_SSID" password "Your_Password"
it's stuck for me. What should it look like? For example, I enter my SSID and password but nothing works, what a load of crap. Adios.

**Comment by Dark-Noir** [comment]
+1 Same issue here.

Here is my work around, skip the step, where you should reconnect to your WiFi.
Reboot the device and do it again manual via the pad screen (to get SSH access again).
After you finished the whole guide for installing KlipperScreen etc., reboot (it will take long), then delete all saved WiFi's (if exist) via the screen and then connect to your WiFi  (also via the screen).
Don't forget to disable WPA3 on your router (even WPA2+3 does not work here).
[See my post.](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/discussions/187)

**Reply by SirVaderx** [reply]
Hello,

So then I'll work my way up to here.
Wait until KlipperScreen show a connection error message on the Speeder Pad and go to Menu --> Network and reconnect it to your WiFi network.

Enter this commands (one at time):

sudo systemctl -q disable dhcpcd 2> /dev/null
sudo systemctl -q stop dhcpcd 2> /dev/null
sudo systemctl enable NetworkManager
sudo systemctl -q --no-block start NetworkManager

Then I restart the speeder pad and look for my WiFi on the speeder pad. That's the right thing to do.


I then ignore this work and skip it, that's the right thing to do
Enter this command to display the list of available WiFi networks:

nmcli device wifi list
Capture d’écran 2024-05-31 à 12 18 16
And connect to your network with this command by replacing Your_SSID and Your_Password:

nmcli device wifi connect "Your_SSID" password "Your_Password"
You can check if you are properly connected with this command:

nmcli device status
Capture d’écran 2024-05-31 à 12 20 43
If all is ok, enter this command to download the second installer script:

curl -L https://raw.githubusercontent.com/Guilouz/Klipper-Flsun-Speeder-Pad/main/Downloads/sp_installer2.sh -o /home/pi/sp_installer2.sh
Then this command to start the script:

sudo bash /home/pi/sp_installer2.sh
